tests/arrays/chunk.test.js

```diff
@@ -0,0 +1,19 @@
+const chunk = require('../../src/arrays/chunk');
+
+describe('chunk', () => {
+  test('creates an array of elements split into groups the length of size', () => {
+    expect(chunk(['a', 'b', 'c', 'd'], 2)).toEqual([['a', 'b'], ['c', 'd']]),

```

Comment:
I suppose lodash methods and methods implemented by your side should have similar result.
Could you please use smth like 
`expect(chunk(['a', 'b', 'c', 'd'], 2)).toEqual(_.chunk(['a', 'b', 'c', 'd'], 2)),`